docs/iot-devkit/devkit-ota.md

```diff
@@ -0,0 +1,165 @@
+
+![IoT Workbench: Cloud](media/iot-workbench-cloud.png)
+
+Select **Azure Provision**.
+
+![IoT Workbench: Cloud -> Provision](media/iot-workbench-cloud-provision.png)
+
+Then VS Code guides you through provisioning the required Azure services.
+
+![IoT Workbench: Cloud -> Provision steps](media/iot-workbench-cloud-provision-steps2.png)
+
+The whole process includes:
+* Select an existing IoT Hub or create a new IoT Hub.
+* Select an existing IoT Hub device or create a new IoT Hub device. 
+
+## Config IoT Hub Connection String
+
+1. Switch the IoT DevKit into **Configuration mode**. To do so:
+
+   - Hold down button **A**.
+   - Push and release the **Reset** button.
+
+2. The screen displays the DevKit ID and 'Configuration'.
+
+	![IoT DevKit Configuration Mode](media/devkit-configuration-mode.png) 
+
+3. Open the command palette and select **IoT Workbench: Device**.
+
+	![IoT Workbench: Device](media/iot-workbench-device.png)
+
+4. Select **Config Device Settings**.
+
+	![IoT Workbench: Device -> Settings](media/iot-workbench-device-settings.png)
+
+5. Select **Select IoT Hub Device Connection String**.
+
+	![IoT Workbench: Device -> Connection string](media/iot-workbench-device-string1.png)
+
+   This sets the connection string that is retrieved from the `Provision Azure services` step.
+
+6. The configuration success notification popup bottom right corner once it's done.
+
+    ![IoT DevKit Connection String OK](media/iot-workbench-connection-done.png) 
+
+## Build and upload the device code
+
+1. Open the command palette and select **IoT Workbench: Device**, then select **Device Upload**.
+
+	![IoT Workbench: Device -> Upload](media/iot-workbench-device-upload.png)
+
+2. VS Code then starts verifying and uploading the code to your DevKit.
+
+	![IoT Workbench: Device -> Uploaded](media/firmware-ota/iot-workbench-device-uploaded.png)
+
+3. The DevKit reboots and starts running the code.
+
+## Test the project
+In order to trigger firmware update event on DevKit, you need to use Azure portal to set automatic device management configuration.
+
+1. In the [Azure portal](https://portal.azure.com), go to your IoT hub.
+
+2. In Section **AUTOMATIC DEVICE MANAGEMENT**, select **IoT device configuration**.
+
+3. Select **Add Configuration** and create a new configuration.
+
+	![IoT Hub: Add Configuration](media/firmware-ota/iothub-add-configuration.png)
+
+4. Give your OTA configuration a unique name, for example, "ota-firmware-update", and add labels to help track our configurations if necessary. Select **Next** to move to next step.
+
+	![IoT Hub: Set Name](media/firmware-ota/iothub-configuration-naming.png)
+
+5. Specifies the target content to be set in targeted device twins. Set **Device Twin Path** as "properties.desired.firmware", and set **Content** in the following JSON format:
+
+	```json
+	{
+		"fwVersion": "1.3.7.56",
+		"fwPackageURI": "https://azureboard2.azureedge.net/prod/devkit-firmware-1.3.7.56.bin",
```

Comment:
IoT devkit will be updated to 1.4.0 soon. Will devkit-firmware-1.3.7.56.bin still work?